apps/explorer/lib/explorer/chain/import/runner/celo_accounts.ex

```diff
@@ -62,14 +62,26 @@ defmodule Explorer.Chain.Import.Runner.CeloAccounts do
     {:ok, accounts}
   end
 
+  defp handle_dedup([head | lst]) do
+    Enum.reduce(lst, head, fn %{attestations_requested: req, attestations_fulfilled: full}, acc ->
+      acc
+      |> Map.put(:attestations_requested, req + acc.attestations_requested)
+      |> Map.put(:attestations_fulfilled, full + acc.attestations_fulfilled)
+    end)
+  end
+
   @spec insert(Repo.t(), [map()], Util.insert_options()) :: {:ok, [CeloAccount.t()]} | {:error, [Changeset.t()]}
   defp insert(repo, changes_list, %{timeout: timeout, timestamps: timestamps} = options) when is_list(changes_list) do
     on_conflict = Map.get_lazy(options, :on_conflict, &default_on_conflict/0)
 
     uniq_changes_list =
       changes_list
       |> Enum.sort_by(& &1.address)
-      |> Enum.dedup_by(& &1.address)
+      |> Enum.group_by(& &1.address)
+      |> Map.values()
+      |> Enum.map(fn a ->
```

Comment:
Can't you just do `Enum.map(&handle_dedup/1)`?

Reply:
Yeah, that's better.